src/Gaufrette/Adapter/Phpseclib/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "gaufrette/phpspeclib-adapter",
+    "type": "library",
+    "require": {
+        "php": ">=5.6",
+        "phpseclib/phpseclib": "^2.0",
+        "knplabs/gaufrette": "0.4.*@dev"
+    },
+    "require-dev": {
+        "phpspec/phpspec": "^3.1",
+        "phpunit/phpunit": "3.7.*"
```

Comment:
3.7 ? Seriously ? This is unmaintained since 5 years at least.

Use ``^4.8.35 || ^5.7`` instead (and then work on switching to the namespaced test case and add `` || ^6`` in the condition)